### ISSUE TITLE
Standardize office names

### ISSUE THREAD

**Issue by dwillis** [openelections/openelections-data-ga#211]
Hey @ATLJoeReed, I'd like to make some standardizations of office names, particularly U.S. House and U.S. Senate. That would impact lots of files - does that work for you?

**Comment by ATLJoeReed** [comment]
Yes I can make this happen. Do we have a list of standardized office names? I can make sure all are correct as I roll through them.

**Comment by dwillis** [comment]
@ATLJoeReed Sure! For Georgia, it would be:

* Governor
* Lieutenant Governor
* Secretary of State
* Attorney General
* Commissioner of Agriculture
* Commissioner of Insurance
* State School Superintendent
* Commissioner of Labor
* U.S. Senate
* U.S. House
* State Senate
* State House

Thanks!

**Comment by ATLJoeReed** [comment]
I have 2000 - 2010 corrected and will get the rest in the next few days.
